client/my-sites/plans/jetpack-plans/jetpack-complete-page/style.scss

```diff
@@ -1,68 +0,0 @@
-@import "calypso/assets/stylesheets/shared/mixins/breakpoints";
-@import "@automattic/onboarding/styles/mixins";
-@import "@wordpress/base-styles/breakpoints";
-
-.layout.is-section-jetpack-connect:not(.is-woocommerce-core-profiler-flow) {
-	background-color: #f6f6f6;
-	#content.layout__content {
-		padding: 44px 17px 26px 18px;
-		@include breakpoint-deprecated( ">960px" ) {
-			padding-top: 35px;
-		}
-	}
-}
-
-#notices.global-notices {
-	top: 82.1px;
-}
-
-// The only way to do "mobile first" responsive layouts in Calypso is using (using the build in @inclides & @mixins) is to use:
-// @include breakpoint-depreciated()???
-// @mixin break-large (medium, small, etc), they all utilize @media (max-width: {size}) which is "desktop first".. Hmmm?
-// vs min-width media queries, which is "mobile first" responsive design. May have to create a P2 about this..
-@include breakpoint-deprecated( "<960px" ) {
-	#notices.global-notices {
-		top: 44px;
-		right: 17px;
-		left: 18px;
-	}
-	.notice {
-		width: calc(100% - 35px);
-	}
-}
```

Comment:
Is there a chance these styles were still needed? They seem like they could apply outside of the Jetpack Complete page that we're deleting

Reply:
Good question. The answer, however, is no: these styles would be loaded only after this page was loaded, and even if they were cluttering the global style scope, there was no chance this would happen since the related route was not really used.